### PR TITLE
refractor server

### PR DESCRIPTION
**Pull request by tonykijak**

### CODE REVIEW

**Comment by tonykijak** [comment]
@ArthurHovanesian @SamMahani @slattouf95 Can you please review my PR?

File: server/index.js [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+});
+
+app.route('/api/rooms/:id')
+  .get((req, res) => {
+    res.send('this is a get request');
+  })
+  .post((req, res) => {
+    res.send('this is a post request');
+  })
+  .put((req, res) => {
```
Comment (slattouf95) [inline server/index.js]: I wouldn't choose the same url name for all requests

**Review by slattouf95 — COMMENTED** [review]
I would create a router file for different request and different url requests for each

**Comment by tonykijak** [comment]
@slattouf95 I updated my endpoints to align with best practices

**Pull request merged**